Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorInstance.cpp

```diff
@@ -652,6 +652,8 @@ namespace AZ::Render
 
             m_meshHandle = AZStd::make_shared<MeshFeatureProcessorInterface::MeshHandle>(
                 m_meshFeatureProcessor->AcquireMesh(meshDescriptor, materials));
+
+            m_meshFeatureProcessor->SetIsAlwaysDynamic(*m_meshHandle, true);
```

Comment:
nit: If you add m_isAlwaysDynamic to the MeshHandleDescriptor, then you can set that on the descriptor without needing to make a separate call.